catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -48,10 +48,11 @@
             description="Enables versioning of both metacards and content."/>
     </OCD>
 
-    <OCD name="Source Poller" id="ddf.catalog.util.impl.SourcePoller">
-        <AD name="Interval" id="interval" type="Integer"
+    <OCD name="Source Status Source Poller Runner" id="sourceStatusSourcePollerRunner">
```

Comment:
✏️ This name reads funny to me. Why did it change from Source Poller? I think that name is less confusing. Or maybe Source Status Poller

Reply:
I changed this because the property is now for a `SourcePollerRunner`, not a `SourcePoller`, and there are now two different `SourcePollerRunner`s. I agree though that it's confusing for the user. I'm fine switching it back to just Source Poller.